kubectl-cloudflow/deploy/deploy.go

```diff
@@ -350,16 +360,15 @@ func findSecretName(spec *domain.CloudflowApplicationSpec, streamletName string)
 	panic(fmt.Errorf("Could not find secret name for streamlet %s", streamletName))
 }
 
-func createSecret(appID string, name string, data map[string]string, cloudflowOperatorOwnerReference metav1.OwnerReference) *corev1.Secret {
+func createSecret(appID string, name string, data map[string]string) *corev1.Secret {
 	labels := domain.CreateLabels(appID)
 	labels["com.lightbend.cloudflow/streamlet-name"] = name
 	secret := &corev1.Secret{
 		Type: corev1.SecretTypeOpaque,
 		ObjectMeta: metav1.ObjectMeta{
-			Name:            name,
-			Namespace:       appID,
-			Labels:          labels,
-			OwnerReferences: []metav1.OwnerReference{cloudflowOperatorOwnerReference},
+			Name:      name,
+			Namespace: appID,
```

Comment:
appID has a value of a namespace?

Reply:
we deploy apps in the namespace = appID ..

Reply:
oh yes right.